### ISSUE TITLE
Canada FIP Image in Header should lead to Government of Canada homepage

### ISSUE THREAD

**Issue by Moro-Code** [cds-snc/platform-forms-client#629]
For consistency of navigation, when clicking on the Government of Canada image in the header, the user should be redirected to the canada.ca homepage as opposed to the GC Forms homepage

Finding added from the [IDRC audit](https://docs.google.com/document/d/1PXLy_kvXF8GAn85b7iUBpt1sOhzDcNh7/edit)

Acceptance Criteria 
- [x] Government of Canada image in header when clicked navigates the user to the canada.ca homepage
- [x] in the header, the alt text description is “navigate to the home page”. While it is clear from the image that this refers to the Government of Canada Home page, the alternative text description does not provide equivalent specificity. In addition, the words “navigate to” are not necessary since this is clear from the element’s functionality (link). “Government of Canada Home page” would be a more clear and concise alt text description.

**Comment by jeberhardt** [comment]
I linked directly to the `/en.html` and `/fr.html` pages on `canada.ca`.
